aws_lambda_powertools/utilities/batch/base.py

```diff
@@ -634,7 +639,8 @@ async def _async_process_record(self, record: dict) -> Union[SuccessResponse, Fa
                 result = await self.handler(record=data)
 
             return self.success_handler(record=record, result=result)
-        except ValidationError:
-            return self._register_model_validation_error_record(record)
-        except Exception:
+        except Exception as exc:
+            if exc.__class__.__name__ == "ValidationError":
+                return self._register_model_validation_error_record(record)
+
```

Comment:
I wonder if the same solution should be applied here @heitorlessa

Reply:
definitely -> done: https://github.com/awslabs/aws-lambda-powertools-python/pull/2157/commits/78284f72371eb8cc5f4b9e55a708a8c298679039

Reply:
Thanks!

Reply:
Yes there are 2 places